templates/default/autolock.erb

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+# This script is launched at the login of the user "kiosk"
+# to detect its inactivity and restart the session
+
+# Settings
+# IDLE_TIME: Idle time before locking, in MINUTES
+# LOCK_CMD : program used to lock. Killing session.
+# NOTIFY   : notify this many SECONDS before locking.
+# NOTIFIER : program used to notify.
+
+IDLE_TIME=<%= @timeout %>
+LOCK_CMD="<%= @command %>"
+NOTIFY="15" # 15 seconds before locking
+NOTIFIER="/usr/bin/zenity --warning --text \"<%= @notification %>\" --timeout 5"
```

Comment:
It would be nice if notification if only shown when admin has set a text for it. And no notifications if that filed is empty.

Reply:
Done